svelte-responsive-layout/src/TopNav.svelte

```diff
@@ -0,0 +1,122 @@
+<script>
+  import ToggleIcon from './icons/ToggleIcon.svelte';
+
+  export let items;
+
+  let isActive = false;
+
+  function handleToggleMenu() {
+    isActive = !isActive;    
+  }
+
+  function handleCloseToggleMenu() {
+    isActive = false;    
+  }
+</script>
+
+<style type="text/postcss">
+  .top-nav {
+    color: var(--secondary-color);
+
+    &-toggler {
+      display: flex;
+      align-items: center;
+      height: 24px;
+      width: 24px;
```

Comment:
Recommended buttons size on mobile is at least 44x44. Here, you make the button 44x44 and then decrease the SVG size to not look ugly.